package.json

```diff
@@ -25,24 +25,32 @@
     "enzyme": "^3.11.0",
     "express": "^4.17.1",
     "grunt": "^1.3.0",
-    "jest": "^26.6.3",
     "mongoose": "^5.11.12",
     "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "sass": "^1.32.5",
-    "supertest": "^6.1.3"
+    "react-dom": "^17.0.1"
```

Comment:
Probably not, I've left it for now to avoid touching too many front end things when its technically "working" as is. Will likely take these dependencies out the in future.